docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -279,6 +279,50 @@ from Optimum Intel instead of NNCF to compress the model to INT8_ASYM.
       results = pipe(phrase)
       print(results)
 
+.. _mxfp4_support:
+
+**Microscaling (MX) Quantization** method has been introduced to enable users
+to quantize LLMs with a high compression rate at minimal cost of accuracy.
+The method helps maintain model performance comparable to that of the conventional
+FP32. It increases compute and storage efficiency by using low bit-width
+floating point and integer-based data formats:
+
++---------------+-----------------+----------------------------+
+|  Data format  |  Data type      | Description                |
++===============+=================+============================+
+| MXFP8         | | FP8 (E5M2)    | | Floating point, 8-bit    |
+|               | | FP8 (E4M3)    | | Floating point, 8-bit    |
++---------------+-----------------+----------------------------+
+| MXFP6         | | FP6 (E3M2)    | | Floating point, 6-bit    |
+|               | | FP6 (E2M3)    | | Floating point, 6-bit    |
++---------------+-----------------+----------------------------+
+| **MXFP4**     | **FP4 (E2M1)**  | **Floating point, 4-bit**  |
```

Comment:
I would let @yury-gorbachev comment if we should expose this in the guides that we have on model optimization topics. Now, MXFP4 does not have any benefits compared types (e.g. INT4, NF4) that we have.

Reply:
If this should not be exposed too much, should we include it in release notes? 
We were told it is even part of marketing messaging.

So, my proposition is to move all this information to a separate article under Weight Compression. This way we have the requested description in docs but at the same time, we do not expose it anywhere beside the CPU article. We can even add some disclaimers stating that it is experimental or early adoption. What do you think?

Reply:
I have moved the aforementioned info to a separate article, as suggested.